ios/scaffold/AppDelegate.swift

```diff
@@ -76,6 +76,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
     }
 
+    func application(application: UIApplication, continueUserActivity userActivity: NSUserActivity, restorationHandler: ([AnyObject]?) -> Void) -> Bool {
+        if let astroViewController = astroViewController {
```

Comment:
Given that we just need to hand off the `userActivity` here. What do you think about making this more concise?
```swift
return astroViewController?.continueUserActivity(userActivity) ?? false 
```